tools/cellprofiler/measure_object_size_shape.xml

```diff
@@ -41,29 +41,29 @@ with open(input_pipeline) as fin:
     module_count = int(v)
     new_count = module_count + 1
     lines[4] = k + ":%d\n" % new_count
-    with open("output", "w") as f:
-        f.writelines(lines)
-        f.write(writemoss())
+    with open("output.cppipe", "w") as f:
+       f.writelines(lines)
+       f.write(writemoss())
 
-f.close()
-        </configfile>    
+    f.close()
+        </configfile>
     </configfiles>
-    
+
     <inputs>
-      <expand macro="input_pipeline_macro" />
-      <repeat name="rpt_object" title="Add another object">
-          <param name="objects_to_measure" label="Select objects to measure" type="text" >
-            <expand macro="text_validator" />
-          </param>
-      </repeat>
-      <param name="zernike_feature" type="select" display="radio" label="Calculate the Zernike features?">
-          <option value="Yes">Yes</option>
-          <option value="No">No</option>
-      </param>
+        <expand macro="input_pipeline_param" />
+        <repeat name="rpt_object" title="new object" min="1">
+            <param name="objects_to_measure" label="Enter the name of the object to measure" type="text">
+                <expand macro="text_validator" />
+            </param>
+        </repeat>
+        <param name="zernike_feature" type="select" display="radio" label="Calculate the Zernike features?" help="Select Yes to calculate the Zernike shape features. Because the first 10 Zernike polynomials (from order 0 to order 9) are calculated, this operation can be time consuming if the image contains a lot of objects. Select No if you are measuring 3D objects with this module.">

```

Comment:
```suggestion
        <param name="zernike_feature" type="select" display="radio" label="Calculate the Zernike features?" help="Select 'Yes' to calculate the Zernike shape features. Because the first 10 Zernike polynomials (from order 0 to order 9) are calculated, this operation can be time-consuming if the image contains a lot of objects. Select 'No' if you are measuring 3D objects with this module.">

```